docs/static/azure-module.asciidoc

```diff
@@ -224,10 +237,61 @@ sure that at least one of these options is different per Event Hub:
 ** storage_connection
 ** storage_container (defaults to Event Hub name if not defined)
 ** consumer_group
+* **Set number of threads correctly.** 
+The number of threads should equal the number of Event Hubs plus one (or more).
+Each Event Hub needs at least one thread. An additional thread is needed to help
+coordinate the other threads. The number of threads should not exceed the number of Event Hubs multiplied by the
+number of partitions per Event Hub plus one. Threads are
+currently  available only as a global setting.
+** Sample: Event Hubs = 4. Partitions on each Event Hub = 3.
+Minimum threads is 5 (4 Event Hubs plus one). Maximum threads is 13 (4 Event
+Hubs times 3 partitions plus one). 
+
+[[azure-module-setup]]
+==== Set up and run the module
+
+Be sure that the `logstash.yml` file is <<configuring-azure,configured correctly>>. 
+
+===== First time setup
+
+Run this command from the Logstash directory:
+
+["source","shell",subs="attributes"]
+-----
+bin/logstash --setup
+-----
+
+The `--modules azure` option starts a Logstash pipeline for ingestion from Azure
+Event Hubs. The `--setup` option creates an `azure-*` index pattern in
+Elasticsearch and imports Kibana dashboards and visualizations. 
+
+===== Subsequent starts
+
+Run this command from the Logstash directory:
+
+["source","shell",subs="attributes"]
+-----
+bin/logstash 
+-----
+
+NOTE: The `--setup` option is intended only for first-time setup. If you include
+`--setup` on subsequent runs, your existing Kibana dashboards will be
+overwritten.
+
+[[exploring-data-azure]]
+==== Explore your data
+When the Logstash Azure module starts receiving events, you can begin using the
+packaged Kibana dashboards to explore and visualize your data. 
+
+To explore your data with Kibana:
 
+. Open a browser to http://localhost:5601[http://localhost:5601] (username:
+  "elastic"; password: "{pwd}")
+. Click *Dashboard*.
```

Comment:
@karenzone the dashboard name is "[Azure Monitor] Overview"

Reply:
@acchen97 Don't you have to click "Dashboard" to see the available dashboards?  Has this changed, or am I misinterpreting you comment?

Reply:
Yes indeed, but let's include the dashboard name so they know where to navigate. :)